pom.xml

```diff
@@ -218,6 +282,7 @@
                         <configuration>
                             <includes>
                                 <include>**/*TestEnterprise.java</include>
+                                <include>**/*EnterpriseIT.java</include>
```

Comment:
I don't see changes we have discussed 

You're right that we can use IT without Test and that's ok for idea regex.
But I'd like to have unification about this question. To have the same things in our every product